### DISCUSSION TITLE
How to convert a C++ WinForm to EXE?

### DISCUSSION THREAD

**Discussion by Panda2340** [community/community#128592] [Programming Help]
Hello,
For the past few days, I have been looking for ways to convert a C++ WinForm made in Microsoft VS2022 to a .exe file, but I haven't found any working methods.
Please help if you can
:)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by rohispace90** [comment]
Hello dear friend;

To convert a C++ WinForm project created in Microsoft Visual Studio 2022 to an executable (.exe):

1. Open your project in Visual Studio.
2. Set the configuration to `Release` mode.
3. Build the project (`Build Solution` or `Ctrl+Shift+B`).
4. Locate the .exe file in `YourProjectFolder\bin\Release\YourProjectName.exe` and test it.

Make sure all necessary DLLs are accessible alongside the executable.

Be lucky.

**Reply by Panda2340** [reply]
Hi,
I figured out how to convert it to EXE a few days ago, but thanks for your reply.

**Reply by ebndev** [reply]
Hi @rohispace90, thanks for taking the time to answer!

@Panda2340 glad to hear you resolved it! Would you mind posting the solution? It would be helpful for other users who might experience this in the future when they search for it 😄

**Reply by Panda2340** [reply]
Sure! I will post it in a bit, as I am on a chromebook, and need to go to my Windows Desktop with VS2022 to do so.

**Reply by Panda2340** [reply]
Sorry, My PC accidentally shut down, and I lost my progress on the post (it was a bit long). I will start making the post again soon, and post it ASAP.

**Reply by ebndev** [reply]
@Panda2340 No worries! It's not urgent or necessary. It would be nice to have but wouldn't want to create more work for you 😄

**Reply by Panda2340** [reply]
It's ok. I want people to be able to follow a guide on here if they are having trouble. :)
